commands.go

```diff
@@ -207,6 +215,10 @@ func (c *ValidateCommand) Run(ctx context.Context, args []string) error {
 		return errGenerate
 	}
 
+	if !c.quiet {
+		fmt.Fprintf(ui.StdWriter, "OK")
```

Comment:
```suggestion
		fmt.Fprintln(ui.StdWriter, "OK")
```

Otherwise shells will look like this when executing the command:

```console
user:~$ til validate foo.brg.hcl
OKuser:~$
```

Same problem if `til validate` is executed in a script.

Reply:
my bad! zsh adds a new line automatically.